File: src/test/java/net/slipp/domain/tag/TaggedHistoryTest.java
package net.slipp.domain.tag;

import static org.hamcrest.CoreMatchers.*;
import static org.junit.Assert.*;

import org.junit.Test;

public class TaggedHistoryTest {

	@Test
	public void create() {
		Long tagId = 1L;
		Long questionId = 2L;
		Long userId = 3L;
		TaggedHistory actual = new TaggedHistory(tagId, questionId, userId, TaggedType.TAGGED);
		assertThat(actual, is(new TaggedHistory(tagId, questionId, userId, TaggedType.TAGGED)));
	}

}

File: src/test/scala/net/slipp/ndomain/tag/TaggedHistoryTest.scala
package net.slipp.ndomain.tag

import net.slipp.domain.tag.{TaggedHistory, TaggedType}
import org.hamcrest.CoreMatchers.is
import org.junit.Assert.{assertNotNull, assertThat}
import org.junit.Test

class TaggedHistoryTest {
  @Test def create() {
    val tagId = 1L
    val questionId = 2L
    val userId = 3L
    val actual = new TaggedHistory(tagId, questionId, userId, TaggedType.TAGGED);
    assertThat(actual, is(new TaggedHistory(tagId, questionId, userId, TaggedType.TAGGED)));
    
    assertNotNull(new TaggedHistory)
  }
}